src/app/searched-news/searched-news.component.scss

```diff
@@ -106,4 +106,27 @@
     .loader{
         grid-column: span 1;
     }
-}
\ No newline at end of file
+}
+
+.img-wrapper {  
+//   width: 400px;
+//   height: 400px;
+  overflow: hidden; 
+}
+
+.inner-img {
+  transition: 0.3s;
+}
+
+.inner-img:hover {
+  transform: scale(1.2);
+}
+
+.effect {
+    transition: 0.3s;
+  }
+  
+  .effect:hover {
+    transform: scale(1.1);
+    z-index: 99;
```

Comment:
Such a high z-index value is not required. It could be simply set to 1 because other cards are at 0 relative to it